test/foundry-tests/PoolManager.t.sol

```diff
@@ -340,6 +341,8 @@ contract PoolManagerTest is Test, Deployers, TokenFixture, GasSnapshot, IERC1155
 
     function testPoolManagerFeeControllerSet() public {
         assertEq(address(manager.protocolFeeController()), address(0));
+        vm.expectEmit(false, false, false, true);
```

Comment:
```suggestion
        vm.expectEmit(true, true, true, true, address(manager));
```

Reply:
Thanks for the suggestion!

In my opinion it makes more sense to specify the first three values as `false` since the purpose of those arguments is to evaluate indexed topics, and the `ProtocolFeeControllerUpdated` event doesn't have any log values encoded as topics. 

Specifying the emitting contract though does seem like it adds value, since it's more comprehensive. I'll update it accordingly!